easybuild/easyconfigs/f/flatbuffers/flatbuffers-23.1.4-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'CMakeNinja'
```

Comment:
@SimonPinches please reinstate the previous author attribution

Reply:
Done - apologies.